index.html

```diff
@@ -42,6 +44,10 @@
             $('#account-display-name').val('');
           });
 
+          $('#delete-account').click(function() {
+            deleteAccount();
+          });
```

Comment:
Heh, yeah, but I didn't want to modify the style/structure of this file too much (most of the focus is on the account-storage logic)